app/src/main/java/com/mathroule/demoapp/Helper.java

```diff
@@ -0,0 +1,10 @@
+package com.mathroule.demoapp;
+
+import java.util.Random;
+
+public class Helper {
+
+    public static String getRandomButtonLabel( ) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/6ba8726f5796dfc005c0166d5e5fe29dcee50c6b/app/src/main/java/com/mathroule/demoapp/Helper.java#L7"></span>’(‘ is followed by whitespace.</td>
    </tr>
  </tbody>
</table>